cloud/export-metrics.mdx

```diff
@@ -17,13 +17,14 @@ Get the corresponding `CLOUD_HOST` for your region and Cloud provider from the t
 
 | Region/CloudProvider | CLOUD\_HOST                          |
 | :------------------- | :----------------------------------- |
-| useast2/aws          | canary-useast2-mgmt.risingwave.cloud |
+| us-east-2/aws        | canary-useast2-mgmt.risingwave.cloud |
 | us-east-1/aws        | prod-aws-usea1-mgmt.risingwave.cloud |
 | us-west-2/aws        | prod-aws-uswe2-mgmt.risingwave.cloud |
 | eu-west-2/aws        | prod-aws-euwe2-mgmt.risingwave.cloud |
 | eu-north-1/aws       | prod-aws-euno1-mgmt.risingwave.cloud |
-| us-central1/gcp      | prod-gcp-usce1-mgmt.risingwave.cloud |
+| ap-southeast-1/aws   | prod-aws-apse1-mgmt.risingwave.cloud |
 | europe-west3/gcp     | prod-gcp-euwe3-mgmt.risingwave.cloud |
+| europe-north1/gcp    | prod-gcp-euno1-mgmt.risingwave.cloud |
```

Comment:
```suggestion
| europe-west-3/gcp     | prod-gcp-euwe3-mgmt.risingwave.cloud |
| europe-north-1/gcp    | prod-gcp-euno1-mgmt.risingwave.cloud |
```

Reply:
In fact, europe-west3 follows the naming of GCP regions: https://cloud.google.com/compute/docs/regions-zones#available